lib/scan.js

```diff
@@ -136,6 +136,13 @@ function Scan (client, ns, set, options) {
    * @member {boolean} Scan#concurrent
    */
   this.concurrent = options.concurrent
+
+  /**
+   * If set to <code>true</code>, the scan will return only those belongs to partitions.
+   *
+   * @member {boolean} Scan#pfEnabled
+   */
+  this.pfEnabled = false
```

Comment:
Should this be `@private`? Else if some client sets `pfEnabled` to `true` directly, but does not set `partFilter`, that would not work, right?